napari/_vispy/vispy_welcome_visual.py

```diff
@@ -0,0 +1,68 @@
+from os.path import dirname, join
+
+import numpy as np
+from imageio import imread
+from vispy.scene.visuals import Text
+from vispy.visuals.transforms import STTransform
+
+from .image import Image as ImageNode
+
+
+class VispyWelcomeVisual:
+    """Welcome to napari visual.
+    """
+
+    def __init__(self, viewer, parent=None, order=0):
+
+        if parent is not None:
+            center = np.divide(parent.canvas.size, 2)
+        else:
+            center = np.array([256, 256])
+
+        logopath = join(dirname(__file__), '..', 'resources', 'logo.png')
+        logo = imread(logopath)
+        logo_colored = np.mean(logo[..., :3], axis=2)
+        logo[..., :3] = np.stack(
+            [logo_colored, logo_colored, logo_colored], axis=2
+        )
+        self._logo = logo
+        self.viewer = viewer
+        self.node = ImageNode(parent=parent)
+        self.node.order = order
+
+        self.node.set_data(self._logo)
+        self.node.cmap = 'gray'
+        self.node.transform = STTransform()
+        center_logo = [center[0] - 100, 1 / 2 * center[1] - 100]
+        self.node.transform.translate = [center_logo[0], center_logo[1], 0, 0]
+        self.node.transform.scale = [
+            200 / self._logo.shape[0],
+            200 / self._logo.shape[0],
+            0,
+            0,
+        ]
+
+        self.text_node = Text(pos=[0, 0], parent=parent)
+        self.text_node.order = order
+        self.text_node.transform = STTransform()
+        self.text_node.transform.translate = [
+            center[0],
+            4 / 3 * center[1],
+            0,
+            0,
+        ]
+        self.text_node.font_size = 25
+        self.text_node.anchors = ('center', 'center')
+        self.text_node.text = (
+            'Drag file(s) here to open \n or \n use File -> Open File(s)'
```

Comment:
Silly details but I really like the unicode arrows in these situations
'Drag file(s) here to open \n or \n use File → Open File(s)'

Reply:
Unfortunately the unicode character written as `'use File \u2192 Open File(s)'` doesn't render right in vispy, but I love the suggestion. 
![image](https://user-images.githubusercontent.com/6531703/97066355-a477dc00-1569-11eb-901b-86a5864361c0.png)

Reply:
ok for baffling reasons it looks like I can create a left arrow, but not right arrow 
![image](https://user-images.githubusercontent.com/6531703/97066421-3bdd2f00-156a-11eb-840a-ee50d25500f1.png)
something seems off with that one too though, I'll keep investigating

Reply:
unfortunately I couldn't figure this out easily, looks like something is off with some clipping, so I've just used a `:`, see screenshot here https://github.com/napari/napari/pull/1721#issuecomment-715664646